src/common/meta/src/instruction.rs

```diff
@@ -218,12 +225,13 @@ mod tests {
             },
             "test/foo",
             HashMap::new(),
+            HashMap::new(),
         ));
 
         let serialized = serde_json::to_string(&open_region).unwrap();
 
         assert_eq!(
```

Comment:
Can we add a compatible serialize/deserialize test?

Reply:
Added.